### DISCUSSION TITLE
Backup su nuovo account

### DISCUSSION THREAD

**Discussion by francescobuonfigliodomotica** [indomus/forum#1866] [HUB: Home Assistant]
Ciao ragazzi,
ho perso tutte le mie password e quindi anche quella di home assistant. Ho approfittato della cosa e ho resettato il sistema inserendo poi un backup basic dove c'erano solo le impostazioni fondamentali. Adesso non mi fa rientrare con i nuovi user (che è sempre la stessa) e password. 
Domanda: nel backup ci sono anche le vecchie credenziali? Se si, c'è qualche possibilità di usare i vecchi backup?
Grazie mille, Francesco

**Comment by snakuzzo** [comment]
trovi tutto sotto la directory `/config/.storage`, ma le credenziali non sono memorizzate in chiaro.
Se vuoi puoi provare a ripristinare i file che trovi sotto quella directory, ma secondo me è uno sbattimento inutile.
Ad ogni modo dai un'occhiata anche [QUI](https://www.home-assistant.io/docs/locked_out/)

**Reply by francescobuonfigliodomotica** [reply]
Perfetto Diego. Grazie come sempre.
